.travis.yml

```diff
@@ -32,6 +48,8 @@ before_install:
   - composer self-update
 
 install:
+  - if [[ "$deps" = "high" ]]; then travis_retry composer --no-interaction --prefer-source update;  fi;
+  - if [[ "$deps" = "low" ]]; then travis_retry composer --no-interaction --prefer-source --prefer-lowest update;  fi;
   - travis_retry composer install --no-interaction --ignore-platform-reqs --prefer-source
```

Comment:
Leaving this here will likely cause issues in the future.

Here's the problem: We removed PHP 5.3 and 5.4 from the testing matrices of the ZF 2.5 components. In some cases, we have updated their `composer.json` files to reflect this, but in the majority, that is still a work in progress.
Once any single one of these specifies a PHP 5.5 dependency, if we do `--ignore-platform-reqs`, that will be ignored, and we'll start getting code installed that will not work for the given PHP version. (I've already seen it happen, which is why I've been actively updating dependencies in the Apigility repos to pin to `~2.4.0` or `>=2.3,<2.5`.)

This repo is an anomaly, in that none of the current ZF components it depends on have non-compatible code in them. However, add just one, and it becomes an issue.

As an example, try doing a `composer require zendframework/zend-view`, commit the change, and push to this PR; it'll break the 5.3 support.